src/clusterfuzz/_internal/bot/fuzzers/syzkaller/runner.py

```diff
@@ -279,13 +279,25 @@ def fuzz(
       engine.FuzzResult
     """
 
-    def _filter_log(content):
-      """Filter unneeded content from log."""
-      result = ''
-      strip_regex = re.compile(r'^c\d+\s+\d+\s')
-      for line in content.splitlines():
-        result += strip_regex.sub('', line) + '\n'
-      return result
+    def _filter_log(content: str) -> str:
```

Comment:
Could we add a test for this filtering functionality?

Reply:
yes we can definitely put a unit test here. I'll place it in the directory you've mentioned :)
`src/clusterfuzz/_internal/tests/core/bot/fuzzers/syzkaller/runners_test.py`

Reply:
unit test added